openvpn/main.tf

```diff
@@ -5,81 +5,104 @@
 # https://raw.githubusercontent.com/hashicorp/best-practices/master/terraform/modules/aws/network/openvpn/openvpn.tf
 #--------------------------------------------------------------
 
-variable "name"               { default = "openvpn" }
-variable "vpc_id"             { }
-variable "vpc_cidr"           { }
-variable "public_subnet_ids"  { }
-variable "ssl_cert"           { }
-variable "ssl_key"            { }
-variable "ssh_username"       { }
-variable "ami"                { default = "ami-5fe36434" }
-variable "instance_type"      { }
-variable "openvpn_user"       { }
-variable "openvpn_admin_user" { }
-variable "openvpn_admin_pw"   { }
-variable "vpn_cidr"           { }
-variable "route_zone_id"      { }
-variable "route_zone_name"    { }
-variable "app_environment"    { }
-variable "ssh_cidr_block"     { }
+variable "name" {
+  default = "openvpn"
+}
+
+variable "vpc_id" {}
+
+variable "vpc_cidr" {}
+
+variable "public_subnet_ids" {}
+
+variable "ssl_cert" {}
+
+variable "ssl_key" {}
+
+variable "ssh_username" {}
+
+variable "ami" {
+  default = "ami-5fe36434"
```

Comment:
Should we remove this default value? Does this AMI even exists at this point?

Reply:
Good question! In fact, it seems nothing uses this module at all? Certainly our vpn terraform config does not.
Will remove the AMI default.